### ISSUE TITLE
ANTs interface should expose the --float parameter

### ISSUE THREAD

**Issue by gerddie** [nipy/nipype#1022]
ANTs provides the option *--float* to run a registration with single floating point accuracy for computations, but this option is not exposed by the nipype interfaces. It would be nice if it did.

**Comment by satra** [comment]
@gerddie - could you please send in a PR? this should be an easy one :)

**Comment by oesteban** [comment]
Is this part of #1100 ?

**Comment by blakedewey** [comment]
No. This is an enhancement to the interface's input spec by allowing computations done as floats (not the default double). As far as I know, this option has been around in ANTs for some time and should not have a backwards compatibility issue. A PR was submitted for this #1024, but never merged. I think it is currently waiting to see if we should specify a Bool or Int trait (I vote bool personally), but they might be waiting for @hjmjohnson to weigh in or it might just have fallen to the side.

**Comment by satra** [comment]
@gerddie - any chance of merging the PR with current master and updating with the bool option. we have used `--float` to indicate such computation for a while now with ants.

**Comment by hjmjohnson** [comment]
There was an obscure ITK bug that only reared it's head under the convoluted use by ANTs.  This has been very recently fixed and incorporated into ANTs (2015-07-05).

Usually it did not present, but depending on the data being registered, and the downsampling choose it could have occurred.  

I'm in need of this feature in nipype along with the new save/load state flags.  I also have a PR in place that may need to be merged with this PR

**Comment by hjmjohnson** [comment]
See Also See #1004 .

Perhaps a completely new interface with all these new changes should be created.

antsRegistration_20150710()  <-- That looks ugly enough to uniquely identify the intended version of ants :)  

I could easily make a new ants tag for 2.2.0 that matches nipype.

antsRegistration --version
NOTE: ./antsRegistration --version
ANTs Version: 2.1.0.post286-g9d5ee
Compiled: Jul  3 2015 17:42:17

**Comment by gerddie** [comment]
Hi, I've updated my branch by pulling in the changes from master.

**Comment by mgxd** [comment]
@gerddie @hjmjohnson is this all set?

**Comment by gerddie** [comment]
Because of moving away from the job where I actually needed this, I haven't had another look ,,,

**Comment by oesteban** [comment]
I think `--float` is currently working ok with ants 2.1.0
